<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_Player</class>
 <widget class="QMainWindow" name="GUI_Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>814</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>350</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Sayonara Player</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../.designer/backup/icons/play.svg.png</normaloff>../../../../../../.designer/backup/icons/play.svg.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>177</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="GUI_Controls" name="controls" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="GUI_PlayerPlugin" name="plugin_widget" native="true">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="GUI_Playlist" name="ui_playlist" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="library_widget" native="true">
       <layout class="QVBoxLayout" name="library_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action_OpenFile">
   <property name="icon">
    <iconset resource="../Resources/Icons.qrc">
     <normaloff>:/Icons/play.svg.png</normaloff>:/Icons/play.svg.png</iconset>
   </property>
   <property name="text">
    <string notr="true">Open File</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+O</string>
   </property>
  </action>
  <action name="action_OpenFolder">
   <property name="icon">
    <iconset resource="../Resources/Icons.qrc">
     <normaloff>:/Icons/sampler.svg.png</normaloff>:/Icons/sampler.svg.png</iconset>
   </property>
   <property name="text">
    <string notr="true">Open Folder</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="../Resources/Icons.qrc">
     <normaloff>:/Icons/power_on.svg.png</normaloff>:/Icons/power_on.svg.png</iconset>
   </property>
   <property name="text">
    <string notr="true">Close</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Dark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string notr="true">F1</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string notr="true">About</string>
   </property>
  </action>
  <action name="action_viewLibrary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true">Library</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+L</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="action_Fullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen </string>
   </property>
  </action>
  <action name="action_devices">
   <property name="text">
    <string>Devices</string>
   </property>
  </action>
  <action name="action_donate">
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="action_logger">
   <property name="text">
    <string notr="true">Logger</string>
   </property>
  </action>
  <action name="action_preferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUI_Playlist</class>
   <extends>QWidget</extends>
   <header>GUI/Playlist/GUI_Playlist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GUI_PlayerPlugin</class>
   <extends>QWidget</extends>
   <header>GUI/Plugins/Generic/GUI_PlayerPlugin.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GUI_Controls</class>
   <extends>QWidget</extends>
   <header>GUI/Player/GUI_Controls.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
